packages/components/src/Illustration/Scene/_docs/Scene.stickersheet.stories.tsx

```diff
@@ -0,0 +1,258 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+
+import {
+  EmptyStatesAction,
+  EmptyStatesInformative,
+  EmptyStatesNegative,
+  EmptyStatesNeutral,
+  EmptyStatesPositive,
+  Information360Upgrade,
+  InformationDemographicFocus,
+  InformationEmergingTrends,
+  InformationEmployeeLifecycle,
+  InformationReportOwner,
+  InformationReportOwnerByRule,
+  InformationTurnoverCalculator,
+  InformationTurnoverForecast,
+  Collaboration,
+  Communication,
+  CompanyValues,
+  ConnectTheDots,
+  CultureLab,
+  TermsAgreement,
+  Programs,
+  PerformanceCompanySettings,
+  EngagementSurveySummaryFemale,
+  EngagementSurveySummaryMale,
+  BrandMomentNewAccountOnboarding,
+  BrandMomentUploadEmployeeData,
+  BrandMomentPositiveOutro,
+  BrandMomentLogin,
+  BrandMomentError,
+  BrandMomentStarterKit,
+  SkillsCoach1On1Meetings,
+  SkillsCoachCoaching,
+  SkillsCoachEmployeeDevelopment,
+  SkillsCoachEssentialFeedback,
+  SkillsCoachEssentialProductivity,
+  SkillsCoachEssentialResilience,
+  SkillsCoachInfluentialCommunication,
+  SkillsCoachLeadingChange,
+  SkillsCoachFeedback,
+  SkillsCoachManagerHub,
+  SkillsCoachProductivity,
+  SkillsCoachRemoteManager,
+  SkillsCoachResilience,
+  SkillsCoachStrategy,
+  SurveyGetStarted,
+  SurveyOverviewClosed,
+} from "../index"
+
+export default {
+  title: "Components/Illustrations/Scene",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: ({ isReversed }) => (
+    <StickerSheet isReversed={isReversed}>
+      <StickerSheet.Body>
+        <StickerSheet.Row>
+          EmptyStatesAction
+          <StickerSheet.Cell width={400}>
+            <EmptyStatesAction />
+          </StickerSheet.Cell>
```

Comment:
I'm sure there's some smart way to iterate, but the way the original did it in `kaizen-legacy` was way too involved for the value we gain